atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/MultiCassandraStartupOrderingEteTest.java

```diff
@@ -68,56 +69,45 @@ public void randomizeKeyspace() throws IOException, InterruptedException {
                 + "/' var/conf/atlasdb-ete.yml");
     }
 
-    @AfterClass
-    public static void resetKeyspace() throws IOException, InterruptedException {
-        EteSetup.execCliCommand("sed -i 's/keyspace: .*/keyspace: atlasete/' var/conf/atlasdb-ete.yml");
-        stopTheAtlasServer();
-        startTheAtlasServer();
+    @After
+    public void restoreCassandra() throws InterruptedException {
+        startCassandraNodes();
     }
 
     @Test
     public void shouldBeAbleToStartUpIfCassandraIsDownAndRunTransactionsWhenCassandraIsUp()
             throws IOException, InterruptedException {
-        stopTheAtlasServer();
-
-        stopCassandraNodes();
-
-        startTheAtlasServer();
-
-        assertNotSatisfiedWithin(40, MultiCassandraStartupOrderingEteTest::canAddTodo);
+        restartAtlasWithDegradedCassandra();
+        assertNotSatisfiedWithin(40, MultiCassandraStartupOrderingEteTest::canPerformTransaction);
 
         startCassandraNodes();
+        assertSatisfiedWithin(60, MultiCassandraStartupOrderingEteTest::canPerformTransaction);
 
-        assertSatisfiedWithin(150, MultiCassandraStartupOrderingEteTest::canAddTodo);
-
-        stopTheAtlasServer();
+        restartAtlasWithDegradedCassandra();
+        Assert.assertEquals(hasQuorum(), canPerformTransaction());
```

Comment:
Not sure what we're testing with this line. Maybe this changed in subsequent PRs?